### ISSUE TITLE
"Missing Prefab" in hierarchy

### ISSUE THREAD

**Issue by bradknox** [EnoxSoftware/VuforiaWithOpenCVForUnitySample#3]
In OS X 10.11 and Unity 5.3.5, I downloaded a zipped clone of the repository, copied the unzipped folder to a project's Assets folder, and then double-clicked on the scene VuforiaWithOpenCVForUnitySample.

The resulting hierarchy has a "Missing Prefab" item. Playing the scene does not show anything other than the magenta rectangle and blue border.

<img width="719" alt="screen shot 2016-08-23 at 5 58 04 pm" src="https://cloud.githubusercontent.com/assets/6486184/17912751/2b41ca6c-695b-11e6-83e4-fa4a24db78fb.png">

**Comment by EnoxSoftware** [comment]
Ｉ have tested using vuforia-unity-mobile-android-ios-4-0-105.unitypackage and Unity 5.0.0f4.
Unfortunately, I have not tested in the latest version.

**Comment by bradknox** [comment]
Thanks for your reply. I'm aware of at least one other having a similar problem in the last week, as evidenced in the comments section of your video tutorial: https://www.youtube.com/watch?v=TnF90ladrOo.

Comparing my hierarchy to the one in your video, there are two missing items. One is "Directional light for UnityChan", which I assume doesn't matter. Another is TextureBufferCamera. Perhaps that's needed for the demo and is the missing prefab?

**Comment by bradknox** [comment]
I was able to get the example running.

I deleted the missing prefab (along with another that was the child of another GameObject). More importantly, I added scripts where they were missing in some script components. To figure out what went where, I closely examined the video.

I think it would be a great service to your customers if you redid your video tutorial and included the import process for _both_ OpenCV for Unity and this repository. Currently the video starts with everything already working, which made it quite challenging on my end. Also, giving a quick tour of the game objects would help.

Lastly, thanks for making this all available.
